lib/mondo/attachment.rb

```diff
@@ -19,8 +19,8 @@ def registration_data
       {
         external_id: self.external_id,
         file_url: self.file_url,
-        file_type: self.file_type, 
+        file_type: self.file_type,
```

Comment:
Looking at this again. I think we are going to need both sets of params. The API seems to take different params when uploading compared to the ones it responds with in the transaction object.